frontend/components/markup-modal/markup-modal-content.tsx

```diff
@@ -12,6 +12,13 @@ const Documentation = styled.div`
   max-width: 800px;
   margin: 35px auto 20px;
   text-align: left;
+
+  mark {
+    padding: 0.2em 0.4em;
+    font-size: 85%;
+    background-color: rgba(27, 31, 35, 0.05);
+    border-radius: 6px;
+  }
```

Comment:
I removed this change, and replaced my documentation with `<code></code>` instead of `<mark></mark>`. It doesn't appears as I wanted like `this` but ok for now.